bibliotheque-universitaire-de-medecine-vancouver.csl

```diff
@@ -46,6 +46,31 @@
       </term>
     </terms>
   </locale>
+  <locale xml:lang="en">
```

Comment:
Several of these (like `in`, `cited`, and `editor`) are already in the English locales, please remove them here.

Reply:
Hi,

No problems, I know there are a lot of styles to review ;-) Thanks for this review.

I adressed the requested changes. The ICMJE asks that _"References should follow the standards summarized in the NLM's Sample References (www.nlm.nih.gov/bsd/uniform_requirements.html) webpage and detailed in the NLM's Citing Medicine, 2nd edition (www.ncbi.nlm.nih.gov/books/NBK7256/)."_ https://www.icmje.org/icmje-recommendations.pdf (p.18 ; ii. Style and Format)

Both references stand that the label for `retrieve` term should be _"Available from"_. e.g. here https://www.nlm.nih.gov/bsd/uniform_requirements.html#item36 Is it ok if I left the label as _"Available from"_ in the style instead of using the default english locale label _"Retrieved"_ ?

I removed the others as requested.

Thanks again!

Reply:
Yes, I just don't want literal copies of the locale file in there. Re-defining terms is fine, that's what the section is for.